lib/ratelimiter.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+var THREE_HOURS = 3* 60 * 60 * 1000;
+
+/**
+ * A rate limiter to be used as an express middleware.
+ *
+ * @param {Object} options
+ * @param {Number} options.limit - Number of requests for normal rate limiter
+ * @param {Number} options.every - Interval of the normal rate limiter
```

Comment:
Could we rename this to "interval" instead of "every"? "every" was not intuitive to me, it would be less so for non-English speakers.